base_import_match/models/base_import.py

```diff
@@ -88,7 +89,7 @@ def _compute_name(self):
                 " + ".join(
                     s.field_ids.mapped(
                         lambda r: (
-                            str(r.field_id.name) +
+                            unicode(r.field_id.name) +
```

Comment:
I double-checked, and after all `name` and `field_id` are required, so no chance of getting `False`. I'll just drop casting.